specifyweb/frontend/js_src/.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
   root: true,
   parser: '@typescript-eslint/parser',
   parserOptions: {
-    project: './tsconfig.eslint.json',
+    project: '/Users/carolinedenis/Desktop/specify7/specifyweb/frontend/js_src/tsconfig.eslint.json',
```

Comment:
Is this change necessary? It works for me without it.
And even if it is necessary, it shouldnt' be pushed as the path you provided only exists on your system